src/components/Banner/Banner.tsx

```diff
@@ -166,7 +166,7 @@ type MessageProps = {
  */
 const BannerMessage: React.FC<MessageProps> = (props: MessageProps) => {
   return props.children ? (
-    <Text color={"inherit"} size="body">
+    <Text color="inherit" size="body">
```

Comment:
Should this be
```suggestion
    <Text variant="inherit" size="body">
```
?

Reply:
Yes, shit, lost confidence in my codemod, thanks for catching